src/thunderbots/software/CMakeLists.txt

```diff
@@ -250,23 +250,19 @@ if (CATKIN_ENABLE_TESTING)
 
```

Comment:
Apologies for a lot of the changes to this file. Ended up reorganizing the files to be alphabetical just to make the merging easier.... but I think it's an overall improvement